templates/main.phtml

```diff
@@ -188,8 +131,8 @@
 
     </div>
 <?php } ?>
-<!--
-<link rel="stylesheet" href="{{ url_for('static', filename='functions.js') }}">
-<link rel="stylesheet" href="{{ url_for('static', filename='prism.js') }}">-->
+
+<script type="text/javascript" src="/static/functions.js"></script>
+<script type="text/javascript" src="/static/prism.js"></script>
```

Comment:
Why are we putting script tags at the bottom of the html? Isn't this a weird hack to get the page to load faster by not loading javascript until the last second? These should probably be in <head>

Reply:
Because that's where they are in all the examples? At least it's consistent.

Reply:
fair enough